src/Storage/Storage.Management/StorageAccount/SetAzureStorageAccount.cs

```diff
@@ -278,7 +283,25 @@ public bool PublishInternetEndpoint
             }
         }
         private bool? publishInternetEndpoint = null;
-        
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Enable Azure Files Active Directory Domain Service Kerberos Authentication for the storage account.",
+            ParameterSetName = AzureActiveDirectoryKerberosForFileParameterSet)]
+        [ValidateNotNullOrEmpty]
+        public bool EnableAzureActiveDirectoryKerberosForFile
+        {
+            get
+            {
+                return enableAzureActiveDirectoryKerberosForFile.Value;
```

Comment:
This line of code may have the potential risk of throwing NullReferenceException since the enableAzureActiveDirectoryKerberosForFile is a nullable field and the parameter itself is not mandatory.

Reply:
```suggestion
                return enableAzureActiveDirectoryKerberosForFile.HasValue ? enableAzureActiveDirectoryKerberosForFile.Value : false;
```

Reply:
Thanks for the suggestion!
Fixed.